evm_loader/cli/src/main.rs

```diff
@@ -1193,6 +1196,77 @@ fn command_neon_elf(
     });
 }
 
+fn command_neon_elf(
+    config: &Config,
+    program_location: Option<&str>,
+) -> CommandResult {
+    if let Some(program_location) = program_location {
+        // Reading program data from file
+        let program_data = read_program_data(program_location).unwrap();
+        let program_data = &program_data[..];
+        read_elf_parameters(config, program_data);
+        Ok(())
+    } else {
+        // Reading program data from account directly
+        if let Some(account) = config.rpc_client
+            .get_account_with_commitment(&config.evm_loader, config.commitment)?
+            .value
```

Comment:
It can be shortened this way:

```
    let account = config.rpc_client
        .get_account_with_commitment(&config.evm_loader, config.commitment)?
        .value.ok_or_else(Err(format!("Unable to find the account {}", &config.evm_loader).into()))?;
 ```
 
 It also would help you avoid a "noodle code"
 
 This recommendation relates to the other `Option` treatments in `command_neon_elf`

Reply:
Looks nice, thanks!

Reply:
Unfortunately, your suggestion is not working, so I'd prefer to leave this code unchanged. Especially taken into account that it was copied from solana =)

Reply:
I'm sorry! Try another one 
https://doc.rust-lang.org/std/option/enum.Option.html#method.ok_or